verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -300,6 +302,8 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
             error: '',
             pageSize: 50,
```

Comment:
this default should be a constant reused in the test to avoid magic numbers

Reply:
I instead added clicks to change the number of rows per page in the test. That has a couple benefits: no longer relies on the default number here, adds test coverage for changing the number of rows per page, and decreases the number of cases needed to add in that test.

Reply:
good call 👍